src/i18n/translations/en.json

```
@@ -291,6 +292,8 @@
   "stopPlaces.noStopPlacesFoundText": "No stop places found.",
   "stopPlaces.numberOfPointsTableHeaderLabelText": "Number of points",
   "stopPlaces.privateCodeTableHeaderLabelText": "Private code",
+  "passingTimes.passingTime": "*Passing time",
```

Ser ut som vi pleier å ha et space mellom * og label-navn.
```suggestion
  "passingTimes.passingTime": "* Passing time",
```